examples/mock-app/README.md

```
@@ -0,0 +1,519 @@
+COPY package*.json ./
+COPY yarn.lock ./
+
+# Install dependencies
+RUN yarn install --production
+
+# Copy source code
+COPY . .
+
+# Build application
+RUN yarn build
+
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)